azure-quantum/azure/quantum/target/ionq.py

```diff
@@ -94,19 +113,16 @@ def estimate_cost(
         self,
         circuit: Dict[str, Any],
         num_shots: int,
-        price_1q: float=0.00003,
-        price_2q: float=0.0003,
-        min_price: float=1.0
+        price_1q: float = None,
+        price_2q: float = None,
+        min_price: float = None
     ) -> CostEstimate:
-        """Estimate the cost of submittng a circuit to IonQ targets.
+        """Estimate the cost of submitting a circuit to IonQ targets.
         Optionally, you can provide the number of gate and measurement operations
         manually.
         The actual price charged by the provider may differ from this calculation.
         
         Specify pricing details for your area to get most accurate results.
```

Comment:
I would add a comment about *the default price is taken from...*

Reply:
Sounds good. Adding.